src/traits/base.php

```diff
@@ -135,6 +135,15 @@ protected function mock() {
 		return $this->mock;
 	}
 
+	/**
+	 * Things needed to run before the test class performs tests.
+	 *
+	 * @return void
+	 */
+	public function setUpBeforeClass(): void { // phpcs:ignore
```

Comment:
Why the `phpcs:ignore`?

We never want to use that annotation inline because it applies both to the line on which it appears, and to the subsequent line. Also, when needed, specific sniffs should be ignored rather than disabling all sniffs

Reply:
Local phpcs was picking it up due to the method not being snake case. However, it looks like that the `phpcs.xml` is excluding that sniff, so I'm removing the ignore altogether.